cmd/mint/root.go

```diff
@@ -16,12 +27,33 @@ var (
 		SilenceErrors: true,
 		SilenceUsage:  true,
 		Version:       config.Version,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			c, err := client.New(client.Config{AccessToken: AccessToken, Host: mintHost})
+			if err != nil {
+				return errors.Wrap(err, "unable to initialize API client")
+			}
+
+			service, err = cli.NewService(cli.Config{APIClient: c, FileSystem: fs.Local{}, SSHClient: new(ssh.Client)})
+			if err != nil {
+				return errors.Wrap(err, "unable to initialize CLI")
+			}
+
+			return nil
+		},
 	}
 )
 
 func init() {
+	// A different host can only be set over the environment
+	mintHost = os.Getenv("MINT_HOST")
+	if mintHost == "" {
+		mintHost = "cloud.rwx.com"
+	}
```

Comment:
You pass in a cloud URL to `mint debug`, right? I wonder if we should infer the host from that URL if MINT_HOST isn't set.